src/frontend/apps/lti_site/components/Transcripts/index.tsx

```diff
@@ -88,47 +102,83 @@ export const Transcripts = ({ transcripts }: TranscriptsProps) => {
     getChoices();
   }, []);
 
+  useEffect(() => {
+    onSelectChange(selectedOption);
+  }, [selectedOption]);
```

Comment:
`onSelectChange` should be dependency, you will have probably to put it in a useCallback or include it directly in useEffect.

Reply:
I get this error in the tests when I add `onSelectChange` in the dependencies : 
`Warning: Maximum update depth exceeded. This can happen when a component calls setState inside useEffect, but useEffect either doesn't have a dependency array, or one of the dependencies changes on every render.`